dnf-behave-tests/features/microdnf/install-nodocs-1.feature

```diff
@@ -0,0 +1,18 @@
+@no_installroot
+Feature: microdnf install command on packages
+
+
+Background:
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
```

Comment:
All the Backgrounds have the steps indented by two more spaces than the scenarios, can you unify it?